.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -63,14 +61,14 @@ steps:
           machineType: "c2-standard-16"
           diskSizeGb: 400
         command: |
-          if [[ -z "${MANIFEST_URL}" ]]; then
+          if [[ -z "$${MANIFEST_URL}" ]]; then
```

Comment:
I'm confused. The switch to `$$` would output the PID, right? So everywhere this is used, the string being checked will be non-null.

Reply:
before any pipeline execution builkite agent uploads the pipeline yaml with `buildkite-agent pipeline upload`. It interpolates the yaml; if it finds `$somethig` it considers it a variable and tries to resolve it during the upload. `${MANIFEST_URL}` will be resolved to `""` (empty string). The `$$` is used to avoid variable substitution. 

* why did it work before?

Reply:
Ah, thanks for explaining. I guess I haven't noticed this because most pipelines have the command logic in separate .sh files.